src/ray/gcs/redis_context.cc

```diff
@@ -665,7 +664,7 @@ std::unique_ptr<CallbackReply> RedisContext::RunArgvSync(
     RAY_LOG(ERROR) << "Failed to send redis command (sync): " << context_->errstr;
     return nullptr;
   }
-  std::unique_ptr<CallbackReply> callback_reply(new CallbackReply(redis_reply));
+  std::unique_ptr<CallbackReply> callback_reply(new CallbackReply(*redis_reply));
```

Comment:
Prefer `make_unique` over `unique_pointer<>(new)` if possible

Reply:
@MortalHappiness can you address these comments in your next PR?

Reply:
Created another PR https://github.com/ray-project/ray/pull/49787